mmdet/core/bbox/transforms.py

```diff
@@ -96,8 +96,8 @@ def bbox2result(bboxes, labels, num_classes):
     """Convert detection results to a list of numpy arrays.
 
     Args:
-        bboxes (Tensor): shape (n, 5)
-        labels (Tensor): shape (n, )
+        bboxes (torch.Tensor or np.ndarray): shape (n, 5)
+        labels (torch.Tensor or np.ndarray): shape (n, )
```

Comment:
```suggestion
        labels (torch.Tensor | np.ndarray): shape (n, )
```

Reply:
I find in mmdet/ops/nms/nms_wrapper.py, the docstring in soft_nms use the same way as I wrote. However, the docstring in nms_match use the way you suggested. How to deal with it?

Reply:
I have replaced 'or' to '|' in these docstring